photon-client/src/main/scala/com/linkedin/photon/ml/Params.scala

```diff
@@ -99,6 +99,7 @@ class Params extends PalDBIndexMapParams {
   /**
    * Whether to enable the optimization tracker, which stores the per-iteration log information of the running optimizer
    */
+  @Deprecated
   var enableOptimizationStateTracker: Boolean = true
```

Comment:
Do we need this deprecated field?

Reply:
I think so - see my reply to another one of your comments.